src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export function Person({ person }) {}
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section>
```

Comment:
```suggestion
  <section className="Person">
```
you don't have the outline without it 😉